kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -135,6 +135,12 @@ const List: React.FC<Props> = ({
                   orderBy={orderBy}
                   setOrderBy={setTopicsOrderBy}
                 />
+                <SortableColumnHeader
+                  value={TopicColumnsToSort.REPLICATION_FACTOR}
+                  title="Replication Factor"
+                  orderBy={orderBy}
+                  setOrderBy={setTopicsOrderBy}
```

Comment:
Pls add setTopicReplicationFactor action. We also need to add td to show replicationFactor from topic object